templates/headless/web-app.instructions.hbs

```diff
@@ -3,16 +3,16 @@ This is a headless skeleton for a hApp.
 To build the app you can run
 
   cd {{app_name}}
-  npm install
-  npm run build:happ
+  {{(package_manager_command package_manager "install" null)}}
```

Comment:
~~it doesn't look like the headless one gets a `package.json` for this command to run against. Should it? it'd be convenient to have the `build:happ` command etc.~~

Wait, I'm guessing that it uses the `package.json.hbs` template from the `generic` template?

Reply:
Yes, it inherits from the generic template